_posts/2019-04-09-sr2019-tlc-crowned-champion.md

```diff
@@ -104,6 +104,7 @@ which will compete against each other in the final competition. They are
 supported by volunteer mentors, and software to assist them in programming
 their robots is provided.
 
+High resolution photographs of the event have been uploaded to our [Google Photos Album](https://photos.app.goo.gl/8vtBZiuM8FXSf79Z8).
```

Comment:
We should put a blank line under this so it puts the next sentence onto a new line.

Reply:
96165fb moves it to a more prominent location, as well as onto its own line.